src/list.html

```diff
@@ -100,6 +100,8 @@ <h2 class="subtitle">
               <li><a href="http://slack.memphistechnology.org/" target="_blank">memtech</a> - Memphis technical community chat.</li>
               <li><a href="https://nashdev.com" target="_blank">NashDev</a> - A network for Nashville developers.</li>
             </ul>
+            <ul><h3>Texas</h3></ul>
+              <li><a href="http://slack.dfwstartupcommunity.com/" target="_blank">DFWStartupCommunity</a></li>
```

Comment:
This needs to be wrapped by `<ul>` and `</ul>` tags.

Reply:
Thanks. I resolved these issues on my editor... but now it seems like I can't remember how to get it back to here.. (sigh). I'm working in ATOM which has everything GitHub integrated into it. I just forget the steps to take.

Reply:
You'll need to `git add` them, then `git commit` them, and finally `git push` them. They will automatically show up on the PR.

Just leave a comment when the changes are made and I'll review. 😃